xdsl/dialects/affine.py

```diff
@@ -83,6 +88,50 @@ def from_region(
         )
 
 
+@irdl_op_definition
+class Store(IRDLOperation):
+    name = "affine.store"
+
+    T = Annotated[Attribute, ConstraintVar("T")]
+
+    value = operand_def(T)
+    memref = operand_def(MemRefType[T])
+    map = opt_attr_def(AffineMapAttr)
```

Comment:
I am confused, why is there this map, whereas the MLIR docu suggests an `indices`-field?

Reply:
Did this change to the most recent MLIR version?

Reply:
It seems so, I added these in such a way to parse the output of the version of MLIR I built on my machine.

Reply:
I see :+1: 
Once the interop tests works, we will know if this aligns with our CI version :sweat_smile:

Reply:
I added an interop test, it passes. I guess one more reason to update to a more recent MLIR

Reply:
Fair, though we will need to add property support for that, which we wanted to postpone as long as possible...